app/models/item_type.rb

```diff
@@ -6,13 +7,20 @@ class ItemType < ActiveRecord::Base
   serialize :allowed_keys, Array
   before_validation -> { self.allowed_keys = allowed_keys.map(&:to_sym) }
 
-  validates :name, presence: true
+  validates :name, :creator_id, presence: true
```

Comment:
Better to validate the presence of the creator itself rather than its ID. It is possible for an item type to point to a creator service which doesn't actually exist.